AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -422,7 +424,7 @@ define([
                                     licenseAndSave(record);
                                 }
                             }
-                        });
+                        })
                     },
 
                     error: function (response) {
```

Comment:
I believe we should stop spinner in case of error too

Reply:
Hi @sidolov. Absolutely, we already have this part 
https://github.com/magento/adobe-stock-integration/blob/develop/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js#L429